janus_server/src/aggregator.rs

```diff
@@ -1784,8 +1798,13 @@ mod tests {
         datastore
             .run_tx(|tx| {
                 Box::pin(async move {
-                    tx.put_task(&TaskParameters::new_dummy(task_id, Vec::new()))
-                        .await
+                    tx.put_task(&TaskParameters::new_dummy(
+                        task_id,
+                        Vec::new(),
+                        Vdaf::Prio3Aes128Count,
```

Comment:
I considered adding a `Fake1` variant to `task::Vdaf` to be used in tests like this that use `aggregator::test_util::fake::Vdaf`. But then you have to add a corresponding value to the enum definition in `db/schema.sql`, and there's no (obvious) way to only do that in test setups.

Reply:
How about we add a second SQL script, an overlay just for testing environments, that runs `ALTER TYPE vdaf ADD VALUE 'FAKE1';`?

Reply:
That would be a pretty good way to do things (though since the fake VDAF is configurable, we'd probably need e.g. `FAKE_SUCCESS`, `FAKE_FAILS_PREP_INIT`, `FAKE_FALES_PREP_STEP`, ... based on the needs of the tests).

But I think we should wait to do this until we have a concrete test case that needs it. Our eventual end-to-end integration test should ideally use a real VDAF IMO.